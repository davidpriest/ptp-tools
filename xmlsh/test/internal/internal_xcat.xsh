# internal_xcat.xsh
# test of xcat


cd $TMPDIR
echo <[<foo><x>bar</x></foo>]> > x1.xml
echo <[<foo><x>spam</x></foo>]> > x2.xml

echo xcat files only
xcat -r -w foo x1.xml x2.xml

echo xcat text element 
xcat -w test -r x1.xml x2.xml
echo xcat xml element
xcat -w <[<test/>]> -r x1.xml x2.xml



