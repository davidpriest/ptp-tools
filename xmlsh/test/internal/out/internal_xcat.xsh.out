xcat files only
<foo>
  <x>bar</x>
  <x>spam</x>
</foo>
xcat text element
<test>
  <x>bar</x>
  <x>spam</x>
</test>
xcat xml element
<test>
  <x>bar</x>
  <x>spam</x>
</test>
